security/s2a/internal/crypter/counter_test.go

```diff
@@ -0,0 +1,109 @@
+package crypter
+
+import (
+	"math"
+	"testing"
+)
+
+type counterTest struct {
+	desc                     string
+	counter, expectedCounter uint64
+	overflow                 bool
+}
+
+func TestCounterInc(t *testing.T) {
+	for _, test := range []counterTest{
```

Comment:
`tc`? To be consistent with the rest of the code. Unless we used `test` everywhere else.

Reply:
It seems I've used a mix of `test` and `tc` throughout. Do we have any preference on `tc` vs `test`? We'll need to change it throughout (in a separate PR).

Reply:
I don't have a preference as long as we're consistent. Let's do the cleanup in a separate PR.